src/main/java/ua/com/javarush/oleksandr/reddit/redditcloneabstract/model/User.java

```diff
@@ -51,6 +38,9 @@ public class User {
 
     private boolean enabled;
 
+    @Column(name = "activation_token")
+    private String activationToken;
```

Comment:
I think it would be better to store the activation token in the other table and add link a user using `user_id`.
Actually we already have a table named `Token` which represents the token entity well. But, you have to make sure that this table isn't used yet in this ticket #21. If so, then create a new table named `activation-token` and inherit the columns from `token` table.